travel-event-host-app/src/components/header/Header.tsx

```diff
@@ -33,32 +40,83 @@ export default function Header() {
           <h1>Backpack</h1>
         </div>
         <div className={styles.authBox}>
-          <label htmlFor='lang' className={styles.langDropdown}>
-            <div className={styles.selection}>
-              {lang}
-              <svg
-                width='1.1em'
-                height='0.8em'
-                viewBox='0 0 18 10'
-                fill='none'
-                xmlns='http://www.w3.org/2000/svg'
+          <Box>
+            <FormControl
+              variant='filled'
+              sx={{
+                m: 1,
+                maxWidth: 90,
+                '& .MuiInputBase-root': {
+                  fontWeight: '400',
+                  fontSize: '1.25rem',
+                  color: theme.palette.primary.thirdColorIceLight,
+                  backgroundColor: theme.palette.primary.secondaryColorDarkBlack,
+                  '&:before': {
+                    borderBottom: 'none',
+                    transition: 'none',
+                  },
+                  '&:after': {
+                    borderBottom: 'none',
+                  },
+                  '&:hover:before': {
+                    borderBottom: 'none',
+                  },
+                },
+                '& .MuiSvgIcon-root': {
+                  color: theme.palette.primary.thirdColorIceLight,
+                },
+                [theme.breakpoints.down(700)]: {
+                  '& .MuiInputBase-root': {
+                    fontSize: '1rem',
+                  },
+                },
+                [theme.breakpoints.down(610)]: {
+                  '& .MuiInputBase-root': {
+                    fontSize: '11px',
+                  },
+                },
+              }}
+            >
+              <Select
+                labelId='selectLanguageDropdown'
+                id='selectLanguageDropdown'
+                value={lang}
+                onChange={handleLanguageOptionChanged}
```

Comment:
so instead handleLanguageOptionChanged
```suggestion
                onChange={handleLanguageOptionChanged}
                (event) => setLang(event.target.value as Language)

```